### PR TITLE
openpgp: ignore unsupported critical packets on cross-signed WoT sigs

### PR DESCRIPTION
**Pull request by maxtaco**
Fixes `keybase id sneak`

### CODE REVIEW

File: openpgp/keys.go [hunk 1]
```diff
@@ -393,7 +393,20 @@ EachPacket:
 			current.UserId = pkt
 		case *packet.Signature:
 
-			// First handle the case of a self-signature. According to RFC8440,
+			// These are signatures but other people on this key. Let's just ignore them
```
Comment (oconnor663) [inline openpgp/keys.go]: s/but/by/

**Comment by oconnor663** [comment]
lgtm

**Comment by dolmen** [comment]
Has it been submitted upstream?

**Pull request merged**